docs/integrations/code-conventions.md

```diff
@@ -885,6 +885,27 @@ return_results(results)
 ```
 **Note:** By default, ignore_auto_extract is set to ```False```.
 
+### Commands in integration with paging
+When dealing with a command in an integration, including paging mechanism (```page``` and ```page_size``` like arguments) convention is:
+- Add to the command ```page```, ```page_size``` and ```limit``` arguments. Where limit argument indicates how many results/incidents/indicators to retrieve from server starting from offset 0, making the number of needed API calls to achieve that limit, and ```page``` and ```page_size``` will be used the normal paging mechanism of the integration.
+
+### Credentials
+When dealing with integrations which require user credentials (such as username/password, API token, etc..) the expected way is to use credentials parameter.
+example for parameter configuration for integration using username and password:
+- display: Username
+  name: credentials
+  type: 9
+  required: true
```

Comment:
(Note that the `\` before the backticks are not needed here it is just due to Github markdown parsing)

```suggestion
In the YML file:
\```
- display: Username
  name: credentials
  type: 9
  required: true
\```

```